### PR TITLE
Add python script to check ssh connection, fixes #157

### LINKED ISSUE #157: sshamigo1 gives no output if command fails
If a new amigo PC is installed (or for other reasons) the keys change.

Then sshamigo2 does not work anymore, ssh amigo@amigo2 also does not work anymore but at least gives some useful output.

Proposal: also show output on such errors

### PR DESCRIPTION
**Pull request by Timple**
Current alias masks errors originating from wrong keys. Happens on replacing amigo computers

Fixes #157

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
Can connect to AMIGO. But not sure if it does what it should do. Trusting you on that.

**Comment by Timple** [comment]
I'll merge right before I'm there again for support ;-)

File: installer/targets/networking/install.bash [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Install config file (mdns4_minimal is normally missing and should be present)
 tue-install-cp nsswitch.conf /etc/nsswitch.conf
+tue-install-cp wait-for-host.py /usr/bin/wait-for-host.py
```
Comment (Rayman) [inline installer/targets/networking/install.bash]: If you are doing this, consider creating a package

**Comment by Timple** [comment]
Fair enough, I'll look into this.

**Comment by Timple** [comment]
Sure, I'm still using it actively at work. But as Rayman said, some troubleshooting commands so people can help themselves would be wiser

**Pull request closed without merging**